distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+====
+
+==== Configuring Files in HOME Directory Hierarchy
+
+Many important configuration settings are stored in the `<${branding}_HOME>` directory.
+
+[IMPORTANT]
+====
+The passwords configured in this section reflect the passwords used to decrypt JKS (Java KeyStore) files.
+Changing these values without also changing the passwords of the JKS causes undesirable behavior.
+====
+
+. In `<${branding}_HOME>/etc/users.properties`, modify the line: `localhost=localhost,group,admin,manager,viewer,system-admin` to be: `<FQDN>=<PASSWORD>,group,admin,manager,viewer,system-admin`
+. Configure `<${branding}_HOME>/etc/system.properties`
+[source,bash]
+----
+#START ${branding} SETTINGS
+# Set the keystore and truststore Java properties
+javax.net.ssl.keyStore=etc/keystores/serverKeystore.jks
+javax.net.ssl.keyStorePassword=<NewPassword>
+javax.net.ssl.trustStore=etc/keystores/serverTruststore.jks
+javax.net.ssl.trustStorePassword=<NewPassword>
+javax.net.ssl.keyStoreType=jks
+
+# Set the global url properties
+org.codice.ddf.system.protocol=https://
+org.codice.ddf.system.hostname=<FQDN>
+org.codice.ddf.system.httpsPort=8993
+org.codice.ddf.system.httpPort=8181
+org.codice.ddf.system.port=8993
+org.codice.ddf.system.rootContext=/services
+
+# HTTPS Specific settings. If making a Secure Connection not leveraging the HTTPS Java libraries and
+# classes (e.g. if you are using secure sockets directly) then you will have to set this directly
+https.cipherSuites=TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_DSS_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA
+https.protocols=TLSv1.1,TLSv1.2
+----
+
+[NOTE]
+====
+Depending on the environment, it may be easier for integrators and administrators to configure ${branding} using the ${admin-console} prior to disabling it for hardening purposes.
+The ${admin-console} can be re-enabled for additional configuration changes.
+====
+
+In an environment hardened for security purposes, access to the ${ddf-branding} ${admin-console} might be denied.
+It is necessary to configure ${branding} (e.g., providers, Schematron rulesets, etc.) using `.config` files or the ${admin-console}.
+Configuration via the Karaf command line console is not supported and may result in configuration errors.
+The OSGi container detects the creation of `.config` files in the `etc/` directory.
+The following sections describe how to configure each ${branding} item using both of these mechanisms.
+A template file is provided for some configurable ${branding} items so that they can be copied/renamed then modified with the appropriate settings.
+
+[WARNING]
+====
+If at a later time the ${admin-console} is enabled again, all of the configuration done via `.config` files is loaded and displayed.
```

Comment:
Verb tenses don't seem right here. Shouldn't it be `If ... is enabled, ... will be loaded and displayed`?